eris/eris-esm/src/modules/Core/listeners/MessageCreateLog.js

```diff
@@ -17,8 +22,12 @@ class MessageCreateLog extends Listener {
         };
     }
 
+    /**
+     * @param {import('eris').Message} message
+     * @param {import('axoncore').GuildConfig} guildConfig
+     */
     execute(message, guildConfig) { // eslint-disable-line
-        if (!message.channel.guild) {
+        if (message.channel instanceof PrivateChannel || message.channel instanceof GroupChannel) {
```

Comment:
couldn't you use channel type instead of instanceof?

Reply:
Wouldn't detect unfortunately

Reply:
rip (imo that definitely a ts flow here)

Reply:
flaw*